code/modules/library/admin.dm

```diff
@@ -0,0 +1,41 @@
+/client/proc/delbook()
+	set name = "Delete Book"
+	set desc = "Permamently deletes a book from the database."
+	set category = "Admin"
+
+	if(!holder)
+		to_chat(src, "Only administrators may use this command.")
+		return
+
+	var/isbn = input("ISBN number?", "Delete Book") as num | null
+	if(!isbn)
+		return
+
```

Comment:
We may want to sanitize this, in case somebody manages to privelige escalate to admin through some other exploit.